src/App.js

```diff
@@ -1,18 +1,54 @@
+import { useState } from "react";
 import "./App.css";
 import Form from "./components/Form";
+import { uid } from "uid";
 
 function App() {
+  const [inputName, setInputName] = useState("");
+  const [booleanGoodWeather, setBooleanGoodWeather] = useState(false);
+  const [inputs, setInputs] = useState({
+    name: inputName,
+    isForGoodWeather: booleanGoodWeather,
+  });
+  const [activities, setActivities] = useState([]);
+
+  function handleAddActivity(inputs) {
+    setActivities([...activities, { id: uid(), inputs }]);
+  }
+
+  console.log("aktis: ", activities);
   function handleSubmit(event) {
     event.preventDefault();
-    const input = event.target.value;
-    console.log(input);
+    handleAddActivity({
+      name: inputName,
+      isForGoodWeather: booleanGoodWeather,
+    });
+
+    // setInputs({ name: inputName, isForGoodWeather: booleanGoodWeather });
+    // handleAddActivity(inputs);
+    // console.log(inputName);
+    // console.log(booleanGoodWeather);
+    // console.log(inputs);
+    // console.log(activities);
+    document.getElementById("App-form").reset();
```

Comment:
you already have access to the form over the event.target

this can therefore be:
```js
event.target.reset();
```